src/main/java/org/broadinstitute/hellbender/tools/recalibration/RecalibrationTables.java

```diff
@@ -1,69 +1,108 @@
 package org.broadinstitute.hellbender.tools.recalibration;
 
 import org.broadinstitute.hellbender.tools.recalibration.covariates.Covariate;
+import org.broadinstitute.hellbender.tools.recalibration.covariates.StandardCovariateList;
 import org.broadinstitute.hellbender.utils.collections.NestedIntegerArray;
 import org.broadinstitute.hellbender.utils.recalibration.EventType;
 
-import java.util.ArrayList;
+import java.util.*;
 
 /**
  * Utility class to facilitate base quality score recalibration.
  */
-
-public final class RecalibrationTables {
-    public enum TableType {
-        READ_GROUP_TABLE,
-        QUALITY_SCORE_TABLE,
-        OPTIONAL_COVARIATE_TABLES_START;
-    }
-
-    private final ArrayList<NestedIntegerArray<RecalDatum>> tables;
+public final class RecalibrationTables implements Iterable<NestedIntegerArray<RecalDatum>>{
     private final int qualDimension;
```

Comment:
I think hardcoding it here would be worse because it would duplicate the logic from QualityScoreCovariate:maximumKeyValue

```
    public int maximumKeyValue() {
        return QualityUtils.MAX_SAM_QUAL_SCORE;
    }
```